draft-ppm-protocol.md

```diff
@@ -548,23 +571,23 @@ struct {
   input share.
 * `payload` is the encrypted input share.
 
-To generate the report, the client begins by encoding its measurements as an
-input for the PPM scheme and splitting it into input shares. (Note that the
-structure of each input share depends on the PPM scheme in use, its parameters,
-and the role of aggregator, i.e., whether the aggregator is a leader or helper.)
-To encrypt an input share, the client first generates an HPKE
-{{!I-D.irtf-cfrg-hpke}} context for the aggregator by running
+To generate the report, the client begins by running the VDAF input-distribution
+algorithm on the public parameter `public_param` and its measurement. This
+transforms the measurement into a pair of input shares, the first intended for
+the leader and the second intended for the helper. To encrypt each input share,
+the client first generates an HPKE {{!I-D.irtf-cfrg-hpke}} context for the
+aggregator by running
 
 ~~~
 enc, context = SetupBaseS(pk,
                           "pda input share" || task_id || server_role)
 ~~~
 
 where `pk` is the aggregator's public key, `task_id` is `Report.task_id` and
-`server_role` is a byte whose value is `0x01` if the aggregator is the leader
-and `0x00` if the aggregator is the helper. `enc` is the encapsulated HPKE
-context and `context` is the HPKE context used by the client for encryption.
-The payload is encrypted as
+`server_role` is a byte whose value is `0x00` if the aggregator is the leader
+and `0x01` if the aggregator is the helper. The bytestring `enc` is the
```

Comment:
```suggestion
`server_role` is a byte whose value is the index of the aggregator in the aggregator list. The bytestring `enc` is the
```

Or some such...

Reply:
Works for me, I thought we were still blocking on this change.